modules/ROOT/pages/lb-mapping-rules.adoc

```diff
@@ -17,6 +17,8 @@ By using proxy rules, you can map a domain or subdomain to one of your Mule appl
 A pattern is a string that defines a template for matching an input text. Whatever is placed into curly brackets (`{   }`) is treated as a variable.
 These variables can contain only letters (a-z) and cannot contain any other characters, such as digits, slashes, etc. The variable values can contain the characters`a-z0-1.&?-_`, but can't contain slashes.
 
+IMPORTANT: Patterns in the application URI are not currently supported.
+
 Suppose you set up a DNS CNAME record from  `example.com` to `_lb-name_.lb.anypointdns.net`. You can then map `app.example.com` to a different deployed CloudHub Mule application name.
```

Comment:
Is it clear to your audience how the second sentence follows logically from the first? It does not to me, because I don't know how the values in the first relate to the different values in the second, but I am not your audience....